arithmetization/src/main/java/net/consensys/linea/bl/TransactionProfitabilityCalculator.java

```diff
@@ -55,8 +54,8 @@ public Wei profitablePriorityFeePerGas(
         .addArgument(conf.estimateGasMinMargin())
         .addArgument(conf.gasPriceRatio())
         .addArgument(conf.verificationGasCost())
-        .addArgument(txSize)
-        .addArgument(compressedSerializedSize)
+        .addArgument(transaction::getSize)
```

Comment:
This value will be different from the earlier logged `txSize` which also added `conf.adjustTxSize()`. Is that expected?

Reply:
good point, I have aligned the fields in the logs